packages/kakoune/src-cs-path-compat.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/src/Makefile b/src/Makefile
+index eef5a2c2..188ef10e 100644
+--- a/src/Makefile
++++ b/src/Makefile
+@@ -71,6 +71,7 @@ else
+ endif
+ 
+ CXXFLAGS += -pedantic -std=gnu++14 -g -Wall -Wextra -Wno-unused-parameter -Wno-reorder -Wno-sign-compare -Wno-address -Wno-noexcept-type -Wno-unknown-attributes -Wno-unknown-warning-option
++CPPFLAGS += -DTERMUX_PREFIX="\"$(TERMUX_PREFIX)\""
+ 
+ all : kak
+ 
+diff --git a/src/shell_manager.cc b/src/shell_manager.cc
+index b92edb4c..df025943 100644
+--- a/src/shell_manager.cc
++++ b/src/shell_manager.cc
+@@ -30,9 +30,7 @@ ShellManager::ShellManager(ConstArrayView<EnvVarDesc> builtin_env_vars)
+ {
+     // Get a guaranteed to be POSIX shell binary
+     {
+-        auto size = confstr(_CS_PATH, nullptr, 0);
+-        String path; path.resize(size-1, 0);
+-        confstr(_CS_PATH, path.data(), size);
++        String path = TERMUX_PREFIX "/bin:" TERMUX_PREFIX "/bin/applets:.";
```

Comment:
You could do `String path = "@TERMUX_PREFIX@/bin:@TERMUX_PREFIX@/bin/applets:.";`, @TERMUX_PREFIX@ is sed'ed before the patch is applied, you can then remove the -DTERMUX_PREFIX flag.